adminservice/src/main/java/nu/yona/server/AdminServiceApplication.java

```diff
@@ -38,8 +40,11 @@
 @SpringBootApplication
 @EnableSwagger2
 @EnableHypermediaSupport(type = HypermediaType.HAL)
+@EnableCaching
 public class AdminServiceApplication
 {
+	@Autowired
+	CacheManager cacheManager;
```

Comment:
Removed the `@Autowired CacheManager cacheManager` from the 3 files, it was not used indeed. Removing it from `CoreConfiguration` doesn't work, it gives:

...
java.lang.IllegalStateException: No CacheResolver specified, and no bean of type CacheManager found. Register a CacheManager bean or remove the @EnableCaching annotation from your configuration.
2015-12-22 06:57:03.027 ERROR 5624 --- [           main] o.s.boot.SpringApplication               : Application startup failed
...